src/routes/init/db/index.ts

```diff
@@ -475,41 +475,41 @@ function getClaimDesc(
 export function consolidate(claimArray: IClaim[]): IClaim[] {
   let myArray = [];
   let unique: string[];
- for(const claims of claimArray) {
-  myArray.push(claims.shortCode);
+  for (const claims of claimArray) {
+    myArray.push(claims.shortCode);
   }
   unique = myArray.filter((v, i, a) => a.indexOf(v) !== i);
 
-  while(unique.length !== 0) {
-  for (const p of claimArray) {
-    for (const q of claimArray) {
-      if (q.shortCode === p.shortCode && q !== p) {
-        p.target.push(q.target[0]);
-        claimArray.splice(claimArray.indexOf(q), 1);
+  while (unique.length !== 0) {
+    for (const p of claimArray) {
```

Comment:
This is a pretty complex function. You should either refactor or comment a little bit